### PR TITLE
Display project in issue list

### PR DESCRIPTION
**Pull request by aceArt-GmbH**
Issues can be added to a project through the issue list, but the assigned project was not displayed

issue_list

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

File: templates/shared/issuelist.tmpl [hunk 1]
```diff
@@ -89,6 +89,11 @@
 							{{svg "octicon-milestone" 14 "mr-2"}}{{.Milestone.Name}}
 						</a>
 					{{end}}
+					{{if .Project}}
+						<a class="milestone" {{if $.RepoLink}}href="{{$.RepoLink}}/projects/{{.Project.ID}}"{{else}}href="{{.Repo.Link}}/projects/{{.Project.ID}}"{{end}}>
```
Comment (zeripath) [inline templates/shared/issuelist.tmpl]:
```suggestion
						<a class="project" {{if $.RepoLink}}href="{{$.RepoLink}}/projects/{{.Project.ID}}"{{else}}href="{{.Repo.Link}}/projects/{{.Project.ID}}"{{end}}>
```

And then we should change less/shared/issuelist.less line 104 from: 

```less
      a.milestone {
```

to 

```less 
      a.milestone,
      a.project {
```

**Review by noerw — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Comment by aceArt-GmbH** [comment]
Can I send a backport for 1.17.X or do you only want bug fixes being back ported?

**Comment by tyroneyeh** [comment]
Could add project data to Issue API?
Thanks

**Pull request merged**